api/src/main/resources/cb_search_api.yaml

```diff
@@ -296,56 +296,88 @@ definitions:
-          table.
+          Code that identifies this criteria. In some vocabularies such as ICD9
+          and ICD10, this code captures the tree hierarchy, e.g. '001.002.003'.
+          Multiple criteria may exist for the same code within a CDR version if
+          a given concept has multiple entries at different locations in the
+          criteria tree (this is common in SNOMED).
+
+          Criteria codes should generally be stable across CDR versions.
         type: string
       name:
         description: description of criteria
         type: string
       count:
-        description: est. count in the cdr
+        description: >
+          Estimated number of participants in a particular CDR version which
+          have a least one event matching this criteria.
         type: integer
         format: int64
       group:
         description: specifies if child or parent
         type: boolean
       selectable:
-        description: specifies if user can search with
+        description: >
+          Whether or not a client can specify this criteria in a search request.
+          Selecting a group implies selecting all criteria contained within that
+          group.
         type: boolean
       conceptId:
         description: >
-          concept ids are ids associated with a specific vocabulary item,
-          in the concept table and allow you to search for that code in
-          its domain table.
+          The OMOP concept id associated with this criteria. May be null if this
+          criteria does not match an OMOP concept, e.g. for intermediate nodes
+          (groups) in the criteria tree. Concept ids are ids associated with a
+          specific vocabulary item in the concept table and allow you to search
+          for that code in its domain table. This may be a source or standard
+          concept ID, depending on the tree for the criteria.
+
+          Similar to a code, multiple criteria may reference the same concept
+          ID due to multiple occurrences of a concept in the criteria tree. In
+          other cases a criteria may share the concept ID of its parent, e.g. in
+          the PPI tree a question and child answer share a concept id.
+
+          Concept IDs should generally be stable across CDR versions.
         type: integer
         format: int64
       domainId:
         description: >
-          clue to determine which tables to search, but these only exist
+          # TODO: More detail - what does this represent exactly? What is the
+          # relationship to the tree type?
+          clue to determine which OMOP tables to search, but these only exist
           for leaves in the tree. Parents don't have domain ids and concept
           id will be used in the case that a parent is selectable.
         type: string
       hasAttributes:
+        # TODO: More detail - what are attributes and what does "are needed" mean?
```

Comment:
If this flag is set, it means that there may be additional details a user might want to add (ie, selecting a lab and entering a value)

Reply:
Thanks - attempted to reword the docs based on this.